indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -192,7 +192,7 @@ <h2>{{ subt }}</h2>
         {% if not document.work.publication_document %}
           {% block verification_notice %}
             <li class="verification-notice">
-              [{% blocktrans %}Note: The original publication document is not available and this content could not be verified.{% endblocktrans %}]
+              [{% blocktrans with no_publication_document_text=document.work.place.settings.no_publication_document_text %}{{ no_publication_document_text }}{% endblocktrans %}]
```

Comment:
I don't think blocktrans is quite right, see here: https://stackoverflow.com/questions/6546933/django-translate-variable-content-in-template. We want it to translate the contents of the variable, not treat it as a translatable string containing a variable.

We should also check if there is no message and not include this notice at all - ie. provide a way to turn it off.